sources/packages/web/src/components/common/NoticeOfAssessmentFormView.vue

```diff
@@ -1,29 +1,37 @@
 <template>
-  <formio formName="noticeofassessment" :data="initialData"></formio>
+  <formio-container formName="noticeOfAssessment" :formData="initialData" />
 </template>
 
 <script lang="ts">
-import { ref, watch } from "vue";
+import { watch, defineComponent, ref } from "vue";
 import { StudentAssessmentsService } from "@/services/StudentAssessmentsService";
 import { AssessmentNOAAPIOutDTO } from "@/services/http/dto";
 
-export default {
+interface NoticeOfAssessment extends AssessmentNOAAPIOutDTO {
+  viewOnly?: boolean;
+}
+
+export default defineComponent({
   props: {
     assessmentId: {
       type: Number,
       required: true,
     },
+    viewOnly: {
+      type: Boolean,
+      required: false,
+    },
   },
-  setup(props: any) {
-    const initialData = ref({} as AssessmentNOAAPIOutDTO);
-
+  setup(props) {
+    const initialData = ref<NoticeOfAssessment>();
     watch(
-      () => props.assessmentId,
+      () => [props.assessmentId, props.viewOnly],
```

Comment:
Can't we use the noaApprovalStatus returned from the API to decide to hide or show the text?

Reply:
is this the answer you are looking for ?
![image](https://user-images.githubusercontent.com/77353155/196248168-23338a92-591a-4c6d-bcca-d39cb3b7fcf7.png)

Reply:
I am not following here. we can talk.